### ISSUE TITLE
GbG Status Screen

### ISSUE THREAD

**Issue by DagAndersonJunk** [dsiekiera/foe-helfer-extension#828]
The GbG Status screen is awesome - the one that shows how many people have made attacks since you last loaded the screen.  Since the tool is read only, could it capture the Sector status as well and show you what Sectors have had increased position?
Dag +4
AvengingGwen +3
Jonathan
Ricsys +2
---
Sector B1O +6
Sector C2S +3

Something like that.  I know we don't have access to who attacked which sector, but this will get close and coordinating GbG sometimes we want people to stop at a certain point on a sector and they don't listen.  For advanced guilds this would be HUUGGGGGEEEEE!

Thanks :)

Dag

**Comment by scibuff** [comment]
Yes, in theory, this is possible, assuming the sector has not changed owners more than once since the last snapshot

**Comment by andrgin** [comment]
In theory this would be possible, but I think it will be a lot of work because we would not only load table with the progress, but also the mapping table ProvinceID => name, ParticipantID => name and track all changes that happen while the map is open.

**Comment by DagAndersonJunk** [comment]
I'm a developer - maybe I should step up, right?  What's the coding stack for the FOE Helper extension?  Is it open source - i.e. I could add to it then submit back for review / publication?

**Comment by dsiekiera** [comment]
Theoretically yes. We sift through it and check that the code is correct. We also take the liberty to adjust this if necessary.

We will reject individual adjustments for your own guild.
Pull requests should always include something useful for all players.

Send us your code via pull request in our beta branch and we will look at it.
